packages/react/src/forms/Radio/Radio.tsx

```diff
@@ -50,8 +50,8 @@ const _Radio = (
         value={value}
         {...rest}
       />
-      <label htmlFor={uniqueId} className={clsx(styles.Radio, className)}>
-        <span className={styles['Radio-dot']} />
+      <label htmlFor={uniqueId} className={clsx(styles.Radio, className)} aria-label={value}>
+        <span className={styles['Radio-dot']}></span>
```

Comment:
What was the reason for making this change? This label is only used to render the radio button and it's dot, and not as the descriptive label for the component.

Adding the label here degrades the screen reader experience a bit as NVDA will announce the label twice.

Before the change, NVDA announces:

```
Mona  radio button  not checked  
Hubot  radio button  not checked  
```

After the change, it announces:
```
Mona mona  radio button  not checked  
Hubot hubot  radio button  not checked  
```

Happy to discuss this further if there's a reason for the change

Reply:
Sounds good, thanks for fixing!